### DISCUSSION TITLE
Java Swing

### DISCUSSION THREAD

**Discussion by Zuba43** [community/community#141878] [GitHub Education]
### Hi everyone,

I have all my Java swing project done and created an executable for windows OS. How do I create one for a Linux based Os?

**Comment by balakrishnan-ultra** [comment]
step 1 : Change the Windows Os file creation path. 
step2 : If any running or compile file .exe, is there change the formate.
step 3 : change the harddisk selection path string.
step 4 : recompile the source code.
